server/app/routes/user/index.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const router = require('express').Router();
+var User = mongoose.model('User');
+
+module.exports = router;
+
+// get all users
+router.get('/', (req, res, next) => {
+	User.find({})
+	.then(user => res.json(user))
```

Comment:
style note: presumably there is more than one user, so I would have used users as the argument/variable.